Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -118,6 +118,32 @@ of removing unused segments after release of internal resources.
   > reflected by queries invoked in subsequent ArangoDB transactions, while
   > in-progress ArangoDB transactions will still continue to return a
   > repeatable-read state.
+  
+  
+  ArangoSearch performs operations in its index based on numerous writer
+  objects that are mapped to processed segments. In order to control memory that
+  is used by these writers (in terms of "writers pool") one can use
+  `writebuffer*` properties of a view.
```

Comment:
This block is rendered as expected, but is indented in source. I guess this works because of the two blank lines above.
Should be okay, but I would remove the leading two spaces from the lines to better reflect the intention.